### PR TITLE
Fix fusermount/fusermount3 detection for libfuse

### PR DESCRIPTION
**Pull request by haampie**

### CODE REVIEW

File: var/spack/repos/builtin/packages/libfuse/package.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ class Libfuse(MesonPackage):
     # Drops the install script which does system configuration
     patch('0001-Do-not-run-install-script.patch', when='~system_install')
 
-    executables = ['^fusermount$']
+    executables = ['^fusermount3?$']
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/libfuse/package.py:35]: I see versions of `libfuse` prior to `3`.  Will this change be appropriate for them should/if a user attempt to install it?

Comment (haampie) [reply var/spack/repos/builtin/packages/libfuse/package.py:35]: It should work, the `?` means that the `3` is optional :)

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/libfuse/package.py:35]: :facepalm: You're correct.  I was thinking of the wrong regex substitution.

**Review by tldahlgren — APPROVED** [review]
LGTM.

**Pull request merged**